### PR TITLE
Add wheel support for aarch64

### PR DESCRIPTION
**Pull request by odidev**
Added wheel support for aarch64.
Related to #57 .

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -121,8 +121,50 @@ jobs:
         name: dist
         path: dist/
 
+  build-linux-aarch64:
```
Comment (elprans) [inline .github/workflows/release.yml:124]: Perhaps move the x86 Linux build into this step as well?

Comment (odidev) [reply .github/workflows/release.yml]: @elprans, I have done the changes as per your suggestion. Please have a look. Thanks in advance.

**Comment by odidev** [comment]
@elprans, could you please have a look at the changes? Thanks in advance.

**Comment by odidev** [comment]
@elprans, could you please have a look at the changes?

**Comment by odidev** [comment]
@fantix, could you please take a look at this PR?

**Comment by odidev** [comment]
@fantix, could you please merge this PR? Thanks

**Comment by elprans** [comment]
Hi @odidev, sorry for taking so long to respond here, and thans for the effort!  We switched to cibuildwheel for builds in 2f57b6b7c2, so aarch64 wheels are now on PyPI.

**Pull request closed without merging**